setup.sh

```diff
@@ -167,8 +167,15 @@ clone_repo() {
 }
 
 clone_kaclone() {
-    echo "Installing ka-clone tool"
-    clone_repo git@github.com:Khan/ka-clone "$DEVTOOLS_DIR"
+    if [ -d "$DEVTOOLS_DIR/ka-clone" ]; then
+        echo "Updating ka-clone tool"
+        cd "$DEVTOOLS_DIR/ka-clone"
```

Comment:
I'm worried that nothing undoes this `cd` -- not sure exactly how that works.  To be safe I'd do:
```
(
    cd ...
    git ...
)
```
Or you could use `git -C "$DEVTOOLS_DIR/ka-clone" ...` and not cd at all.